clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/KubernetesConfiguration.groovy

```diff
@@ -50,4 +51,9 @@ class KubernetesConfiguration {
   KubernetesHealthIndicator kubernetesHealthIndicator() {
     new KubernetesHealthIndicator()
   }
+
+  @Bean
+  KubernetesUtil kubernetesUtil() {
```

Comment:
Now that `KubernetesUtil` gets injected, we will want to update the `deploy` operation to have it autowired as well. Please drop a todo to make those changes and address the other `static` methods. Then we can do that work in a followon PR.